<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1100</width>
      <height>700</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1100</width>
        <height>700</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton
{
    color: #57606f;
    background-color: #ced6e0;
    border-width: 1px;
    border-color: #0097e6;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
	font-size:18px;
	font-family:Kozuka Gothic Pr6N B;
	font-style:bold;
}

QPushButton::hover
{
    border: 2px solid QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);;
	background-color:#f1f2f6;
	color:QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}
QLabel
{
	color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);;
	font-size:18px;
	font-family:Kozuka Gothic Pr6N B;
	font-style:bold;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color:white;
	font-size:15px;
	font-family:Kozuka Gothic Pr6N B;
	font-style:bold;
}</string>
       </property>
       <widget class="QLabel" name="label_logo">
        <property name="geometry">
         <rect>
          <x>27</x>
          <y>13</y>
          <width>171</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: transparent;
border-image:url(:/img_parking/New folder/home.png);
background: none;
border: none;
background-repeat: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_entete">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1100</width>
          <height>70</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color:#f1f2f6;
border-image:url(:/img_parking/entete.png);
/*background: none;*/
border: none;
background-repeat: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>225</y>
          <width>1100</width>
          <height>475</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Kozuka Gothic Pr6N B</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">/*border-image:url(:/img_parking/orange-linear-gradient-1920x1080-c2-ff4500-ff8c00-a-330-f-14.png);
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
/*border-image:url(:/img_parking/orange-linear-gradient-1920x1080-c2-ff4500-ff8c00-a-330-f-14.png);*/
border: none;
background-repeat: none;
 background-color:#f1f2f6;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelAccueil">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1100</width>
          <height>225</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/img_interface/833d042fa9d9e38cea2e687f2fb4b3a4.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_jardin">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>200</width>
          <height>60</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>JARDIN</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img_interface/seeding (1).png</normaloff>:/img_interface/seeding (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>200</width>
          <height>60</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>SECURITE</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img_interface/shield (1).png</normaloff>:/img_interface/shield (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>200</width>
          <height>60</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>PARKING</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img_interface/parking (1).png</normaloff>:/img_interface/parking (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_frigo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>460</y>
          <width>200</width>
          <height>60</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>FRIGO</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img_interface/fridge (1).png</normaloff>:/img_interface/fridge (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_chambre">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>530</y>
          <width>200</width>
          <height>60</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>CHAMBRES</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img_interface/hotel (1).png</normaloff>:/img_interface/hotel (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="Line" name="line_7">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>240</y>
          <width>20</width>
          <height>421</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>70</y>
          <width>180</width>
          <height>50</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>DECONNEXION</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>img_interface/exit.png</normaloff>img_interface/exit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>120</y>
          <width>180</width>
          <height>50</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>QUITTER</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/img_interface/power-button-off.png</normaloff>:/img_interface/power-button-off.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>1000</x>
          <y>12</y>
          <width>41</width>
          <height>47</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButton_8
{
  background-color: transparent;
border-image:url(:/img_parking/Nouveau dossier/round-account-button-with-user-inside (1).png);

border: none;
background-repeat: none;
}
#pushButton_8::hover
{
  border-image:url(:/img_parking/Nouveau dossier/round-account-button-with-user-inside.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>label_2</zorder>
       <zorder>labelAccueil</zorder>
       <zorder>label_entete</zorder>
       <zorder>label_logo</zorder>
       <zorder>pushButton_jardin</zorder>
       <zorder>pushButton_2</zorder>
       <zorder>pushButton_3</zorder>
       <zorder>pushButton_frigo</zorder>
       <zorder>pushButton_chambre</zorder>
       <zorder>line_7</zorder>
       <zorder>pushButton_6</zorder>
       <zorder>pushButton_7</zorder>
       <zorder>pushButton_8</zorder>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
